gunicorn/config.py

```diff
@@ -1836,3 +1845,16 @@ class PasteGlobalConf(Setting):
 
         .. versionadded:: 20.0
         """
+
+class SpawnWorkerSleepTime(Setting):
+    name = "spawn_worker_sleep_time"
+    section = "Worker Processes"
+    cli = ['--spawn-worker-sleep-time']
+    validator = validate_float
+    default = 0.05
```

Comment:
@benoitc This should ensure that we have a default closer to the existing behavior, right?